homeassistant/components/mqtt/models.py

```diff
@@ -7,19 +7,30 @@
 import attr
 
 PublishPayloadType = Union[str, bytes, int, float, None]
+ReceivePayloadType = Union[str, bytes]
 
 
 @attr.s(slots=True, frozen=True)
-class Message:
+class PublishMessage:
```

Comment:
I'm not too sure about the name, maybe `TxMessage` is better?

Reply:
Request / Response?
Send / Receive?

Reply:
You publish MQTT messages, so maybe the pari Publish / Receive is OK?